app/jobs/cache/browser_channels/responses_for_prefix.rb

```diff
@@ -23,6 +23,17 @@ def perform(prefix)
   def generate_brotli_encoded_channel_response(prefix:)
     @site_banner_lookups = SiteBannerLookup.where("sha2_base16 LIKE ?", prefix + "%")
     channel_responses = PublishersPb::ChannelResponseList.new
+
+    parameters = Rewards::Parameters.new.get_parameters
+
+    case parameters
+    when Rewards::Types::ParameterResponse
+      allowed_regions = parameters.custodianRegions
+    else
+      LogException.perform(failure)
```

Comment:
It looks like we're generally using sentry to log errors from the jobs folder.  However, the LogException service will log errors to both sentry and new relic.

Reply:
I've never logged into sentry, @tsmartt do we still use this?  I've only ever looked at newrelic